docs/0.5/examples/gameroom/walkthrough/images/auth_login_bubba1.svg

```diff
@@ -0,0 +1,103 @@
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
0.5 images need to be updated as well

Reply:
I updated the 0.5 images to remove the XML and doctype lines